.github/workflows/hail-search-persistent-volume-snapshot-release.yaml

```diff
@@ -46,13 +46,13 @@ jobs:
         uses: mikefarah/yq@v4.22.1
         with:
           cmd: >
-            yq -i '.global.hail_search.datasetVersions.${{ inputs.reference_genome }}/${{ inputs.dataset_type }} = "${{ inputs.version }}"' charts/broad-seqr/values-${{ inputs.environment }}.yaml
+            yq -i '.global.hail_search.datasetVersions.${{ inputs.reference_genome }}/${{ inputs.dataset_type }} = "${{ inputs.version }}"' charts/${{ inputs.environment }}-broad-seqr/values.yaml
```

Comment:
I think we need a conditional on `${{ inputs.environment }}` so that it's a prefix of `''` if its prod 😢.

Reply:
🙏